docs/reference/docker-support.mdx

```diff
@@ -0,0 +1,263 @@
+:::
+
+### Option `rti-image`
+To run a federated program, an RTI process must run to support it. By default, an image for the RTI is pulled from DockerHub. An alternative image can be specified using the `rti-image` entry.
+
+The default is `"lflang/rti:latest"`, which is available on DockerHub. To instruct the compiler to _not_ pull an image from DockerHub and build the RTI locally, use `rti-image: "rti:local"`.
+
+:::note
+The value of the `builder-base`, `runner-base`, and `rti-image` entry should follow Docker's `<user-name>/<image-name>:<tag-name>` naming convention for image names. Docker will resolve the name and pull the image from your local registry, or, if it cannot be found, from DockerHub.
+:::
+
+## Manually building and running
+The generated executable simply invokes `docker compose up --abort-on-container-failure`, but this might not be what you want. Here are some guidelines for building and running manually.
+If you instead want to build manually after code generation has completed, you can instruct to Lingua Franca compiler to skip building using the `no-build` option in the `docker` target property. More information can be found [here](#option-no-build).
+
+### Using `docker build` and `docker run`
+You can build images and run containers in separate steps. First, change directory to [the location of the `Dockerfile`](#where-to-find-the-docker-configuration-files).
+
+```sh
+docker build -t foo .
+```
+
+This will create a Docker image with tag `foo`. The tag is required to be all lower-case letters. By convention, we advise using the LF source file name, converted to lower case.
+
+You can then use this tag to run the image in a container:
+
+```sh
+docker run -t --rm foo
+```
+
+The `-t` option creates a pseudo terminal, which is necessary for you to see any output produced by your program to `stdout`. If your program also reads from `stdin`, then you will need to give the `-i` option as well, or combine the two as `it`.
+
+The `--rm` option is important. This removes the container upon completion of the run. If you omit this option, the container will continue to exist even after your program has terminated. You can alternatively remove the container after the run using `docker rm`.
+
+If you wish for your program to run in the background, give a `-d` option as well (for "detached"). In this case, you will not see any output from your run.
+
+The above run command can include any supported command-line arguments to the LF program. For example, to specify a logical timeout, you can do this:
+
+```sh
+docker run -t --rm foo --timeout 20 sec
+```
+
+### Manually launching a federation
+In addition to building the images and running each individual container, launching a federation requires a few additional steps.
+
+#### Create a network and launch RTI
+To pull the RTI from DockerHub, run this command:
+```sh
+docker pull lflang/rti:rti
```

Comment:
Not strictly necessary, since `docker compose` will pull missing images. Also not relevant if using `rti:local`. Should be okay to remove

Reply:
This is assuming someone doesn't want to use `docker compose` though, but do stuff manually with `docker build` and `docker run`.

Reply:
Corrected the nesting of titles to reflect my comment. I'm assuming that addresses it.